vendor/engines/projects/app/controllers/projects/global_search_controller_extension.rb

```diff
@@ -0,0 +1,13 @@
+module Projects
+
+  module GlobalSearchControllerExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      searcher_classes.unshift(Projects::GlobalSearch::ProjectSearcher)
```

Comment:
It's arbitrary. It seemed like the case of a search query matching a project name **and** a statement or order number would be rare and ordering wouldn't matter.

But then if this turns back into a `Hash`, injections like this will always be appends.